lib/bugspots/scanner.rb

```diff
@@ -14,8 +14,9 @@ def self.scan(repo, branch = "master", depth = 500, regex = nil)
     end
 
     walker = Rugged::Walker.new(repo)
-    walker.sorting(Rugged::SORT_TOPO | Rugged::SORT_REVERSE)
```

Comment:
> In SORT_REVERSE mode, a walker scans from the initial commit to latest one. I think it would be preferable to scan from latest commit to old one when using depth parameter.

Ok, that makes sense. 

I know we didn't actually apply the default depth before.. but, should we?